app/controllers/posts_controller.rb

```diff
@@ -34,4 +35,15 @@ def show
     @post = Post.find(params[:id])
     @user = User.find(params[:user_id])
   end
+
+  # posts_controller.rb
```

Comment:
[OPTIONAL] Kindly remove this if this comment does not have informative case to use :)